core/deployment/src/main/resources/dev-ui/qwc-qosdk-controllers.js

```diff
@@ -0,0 +1,65 @@
+import { LitElement, html, css, nothing} from 'lit';
+import { JsonRpc } from 'jsonrpc';
+import '@vaadin/grid';
+import '@vaadin/details';
+import '@vaadin/list-box';
+import '@vaadin/item';
+import {columnBodyRenderer} from '@vaadin/grid/lit.js';
+export class ControllersWebComponent extends LitElement {
```

Comment:
Can the data change during runtime ? So can the developer do something while coding that can change this list of controllers ? In that case you can use `QwcHotReloadElement` rather than `LitElement` and that gives you a `hotreload()` method that will be called on hot reload. In that method you can fetch the data again. This mean that the screen will update automatically on hot reload

Reply:
Yes, I still need to support this.

Reply:
Done.